xformers/components/attention/attention_mask.py

```diff
@@ -125,7 +125,6 @@ def shape(self):
         return self.values.shape
 
     def __add__(self, other):
-        assert isinstance(other, type(self))
         return AttentionMask(self.values + other.values, is_causal=False)
```

Comment:
follow the "if it looks like a duck and quack like a duck then it's a duck" python mantra. In any case @fmassa is rewriting this and it will disappear